### PR TITLE
Bug fix resource collection rel hrefs

### LINKED ISSUE #12: Next/Prev hrefs link when using ResourceCollection
Steps to reproduce:

```ruby
client = Groovehq::Client.new

client.tickets.rels[:next].get
# ArgumentError: bad argument (expected URI object or URI string)
# from /Users/ross/.gem/ruby/2.3.0/gems/httparty-0.13.7/lib/httparty/request.rb:55:in `path='

tickets.rels[:next].href
# => nil
```

### PR DESCRIPTION
**Pull request by rossta**
Fixes #12

### CODE REVIEW

File: lib/groovehq/resource.rb [hunk 1]
```diff
@@ -15,11 +15,12 @@ def initialize(client, data)
...
-      @rels = parse_links(links).with_indifferent_access
+      @rels = parse_links(links)
     end
 
     def parse_links(links)
-      (links || {}).each_with_object({}) do |(relation, value), result|
+      rels = ActiveSupport::HashWithIndifferentAccess.new
+      (links || {}).with_indifferent_access.each_with_object(rels) do |(relation, value), result|
         result[relation] = Relation.new(@client, value[:href])
       end
```
Comment (Fodoj) [inline lib/groovehq/resource.rb]: could you try a bit shorter version:

```ruby
(links || {}).each_with_object({}) do |(relation, value), result|
  result[relation] = Relation.new(@client, value[:href])
end.with_indifferent_access
``

**Comment by Fodoj** [comment]
Please merge with master so tests can pass

**Comment by rossta** [comment]
Master has been rebased but travis doesn't seem to be picking up the GROOVEHQ_ACCESS_TOKEN.

**Comment by rossta** [comment]
Looks like [Travis does not support encryption keys for pull requests from forks](https://docs.travis-ci.com/user/pull-requests#Security-Restrictions-when-testing-Pull-Requests)

**Comment by Fodoj** [comment]
Well that kinda sucks :( Seems like I will have to run them myself before releasing new versions and disable them on travis.

**Comment by Fodoj** [comment]
@rossta please rebase again https://github.com/Fodoj/groovehq/pull/14 and then I can merge and release new version with your PRs included, thanks!

**Comment by rossta** [comment]
Rebased.

One idea could be to trigger the integration specs only on master using an env var like `TRAVIS_BRANCH`.

**Pull request merged**